src/com/rcv/Tabulator.java

```diff
@@ -447,7 +447,7 @@ private CandidateStatus getCandidateStatus(String candidate) {
       SortedMap<BigDecimal, LinkedList<String>> currentRoundTallyToCandidates
   ) {
     List<String> eliminated = new LinkedList<>();
-    if (eliminated.isEmpty() && config.isBatchEliminationEnabled()) {
+    if (config.isBatchEliminationEnabled()) {
```

Comment:
This `eliminated.isEmpty()` makes me wonder if there's a bug here, rather than it just being an oversight. Why check this immediately after a new (empty) List is created?

Reply:
Pretty sure this was just an oversight when I was refactoring this code.